### ISSUE TITLE
migrate to nix-community

### ISSUE THREAD

**Issue by Lillecarl** [wamserma/flake-programs-sqlite#14]
Hi!

I've been using your solution for quite awhile now, and other than briefly yesterday it's never blocked my flake updates.

Not that I have any say in it, but a lot of popular and useful nix tools "graduate" to nix-community on github, it increases discoverability and increases collaboration while still not having to maintain the standards of nixos/nixpkgs (which can be quite off-putting for your own projects).

Thoughts? 😄

**Comment by wamserma** [comment]
> I've been using your solution for quite awhile now, and other than briefly yesterday it's never blocked my flake updates.

Great to hear!
 
> Not that I have any say in it, but a lot of popular and useful nix tools "graduate" to nix-community on github, it increases discoverability and increases collaboration while still not having to maintain the standards of nixos/nixpkgs (which can be quite off-putting for your own projects).
> 
> Thoughts? 😄

Donating/Migrating this project to the nix-community repo is definitely something that should eventually happen. As you experienced, there are still some quirks that I'd like to fix before I consider this project mature enough.

**Comment by Lillecarl** [comment]
Thanks for your reply, and thanks again for your solution! 😄

**Comment by wamserma** [comment]
I'll keep this open as a reminder.

**Comment by wamserma** [comment]
As of 26.05 using the lockable HTTP tarball input mechanism is the preferred choice for flakes, which renders this project redundant for most scenarios. It will live on as a niche project outside the nix-community repo.

**Comment by Lillecarl** [comment]
Yeah that's the approach I've been using for a long time now, thanks!
